elastic/beats
# Improve handling of disabled commands in Zookeeper Metricbeat module

## What does this PR do?



## Why is it important?


- Since Zookeeper 3.5.3 , There are few commands in zookeeper will not be enabled by default, So it is important to mention this information in module documentation.
- Also it was noticed we are handling this message in Debug label, So this has been requested mark as ERROR in logging level.
## Checklist



~~- [ ] I have made corresponding change to the default configuration files~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~

## How to test this PR locally
- Clone the beats Repo
- Install the zookeeper using docker-compose 
- Please note that do not whitelist the configuration parameter like 'cons' while installing zookeeper
- Enable the zookeeper module with proper configuration zookeeper.yml file
- Run the metricbeat and you should be seeing a error level logging message saying 'cons' command not whitelisted.

## Related issues


- Closes #20376 

## Screenshot
```
{"log.level":"error","@timestamp":"2022-04-07T18:34:05.050+0530","log.logger":"zookeeper.connection","log.origin":{"file.name":"connection/data.go","file.line":47},"message":"Error while parsing zookeeper 'cons' command no data captured in 'cons is not executed because it is not in the whitelist.'","service.name":"metricbeat","ecs.version":"1.6.0"}
```
